bok_choy/a11y/a11y_audit.py

```diff
@@ -0,0 +1,228 @@
+"""
+Interface for running accessibility audits on a PageObject.
+"""
+import json
+import os
+import requests
+
+from textwrap import dedent
+
+
+class AccessibilityError(Exception):
+    """
+    The page violates one or more accessibility rules.
+    """
+    pass
+
+
+class A11yAuditConfig(object):
+    """
+    The `A11yAuditConfig` object defines the options available in an
+    accessibility ruleset.
+    """
+    def __init__(self, *args, **kwargs):
+        """
```

Comment:
nit: Is pylint requiring a docstring? I think if it'll be empty, don't have one...